packages/runtimes/hmr/src/loaders/hmr-runtime.js

```diff
@@ -96,17 +96,26 @@ if ((!parent || !parent.isParcelRequire) && typeof WebSocket !== 'undefined') {
       !/localhost|127.0.0.1|0.0.0.0/.test(hostname))
       ? 'wss'
       : 'ws';
-  var ws = new WebSocket(
-    protocol + '://' + hostname + (port ? ':' + port : '') + '/',
-  );
+
+  var ws;
+  try {
+    ws = new WebSocket(
```

Comment:
This only throws if the url is invalid (otherwise it's an error event). Does that happen in the webextension case?

Reply:
It seems to throw in Chrome due to CSP errors as well, which is what this tries to avoid logging.